### DISCUSSION TITLE
Installing postgis_sfcgal extension version >= 1.4, ideally based on the latest stable CGAL release (5.5.1)

### DISCUSSION THREAD

**Discussion by swiss-knight** [postgis/docker-postgis#335] [Ideas]
Hello,

I'm wondering if it would be possible to release a Docker image on the hub based on Debian sid (testing) / or the latest Ubuntu (22.04) / or to embed the latest `libboost`, `libcgal-dev` and `libsfcgal1` libraries? Especially for testing the latest PostGIS version.

Why I'm asking?

It's simply for being able to install the `postgis_sfcgal` extension in its version 1.4+ (based on CGAL/BOOST).

Because currently, on Debian bullseye, `libsfcgal1` is only available in version 1.3.9 (https://packages.debian.org/bullseye/libsfcgal1) which makes the `postgis_sfcgal` extension one of the two extensions that are older on Debian than when PostGIS is installed on a native Ubuntu 22.04 OS:

### Native Ubuntu 22.04.1 (jammy)
```
POSTGIS="3.3.2 4975da8"
  [EXTENSION] 
    PGSQL="150"
    GEOS="3.10.2-CAPI-1.16.0"
    SFCGAL="SFCGAL 1.4.1, CGAL 5.3.1, BOOST 1.74.0"   <---- newest
    PROJ="8.2.1"
    LIBXML="2.9.13"
    LIBJSON="0.15"
    LIBPROTOBUF="1.3.3"
    WAGYU="0.5.0 (Internal)"
    TOPOLOGY
```

### Debian 11 (latest official PG15/PostGIS3.3 docker hub image)
```
POSTGIS="3.4.0dev 3.3.0rc2-335-gae95dd219"
  [EXTENSION]
    PGSQL="150"
    GEOS="3.12.0dev-CAPI-1.18.0"
    SFCGAL="1.3.10"                                   <---- older
    PROJ="9.2.0"
    LIBXML="2.9.10"
    LIBJSON="0.15"
    LIBPROTOBUF="1.3.3"
    WAGYU="0.5.0 (Internal)"
    TOPOLOGY
```

On the other hand, `libsfcgal1` 1.4.1-5 is available on sid: https://packages.debian.org/sid/libsfcgal1

See also:  https://dba.stackexchange.com/q/321368

Related PR: https://github.com/postgis/docker-postgis/pull/336

Thank you for your support and for making PostGIS available in Docker!
Warm Regards.

**Comment by ImreSamu** [comment]
I have created an alternative solution: 
* https://github.com/postgis/docker-postgis/pull/337
